src/mc_rbdyn/Robot.cpp

```diff
@@ -157,6 +160,65 @@ void fixSCH(const mc_rbdyn::Robot & robot, mapT & data_, const std::map<std::str
   }
 }
 
+bool VisualToConvex(const std::string & robot,
+                    const std::string & cName,
+                    const std::string & bName,
+                    const rbd::parsers::Visual & visual,
+                    std::map<std::string, mc_rbdyn::Robot::convex_pair_t> & convexes,
+                    std::map<std::string, sva::PTransformd> & collisionTransforms)
+{
+  // Ignore visual types that we cannot easily map to SCH
+  if(visual.geometry.type == rbd::parsers::Geometry::Type::UNKNOWN
+     || visual.geometry.type == rbd::parsers::Geometry::Type::MESH)
+  {
+    return false;
+  }
+  // If we already have a convex with the same name, discard loading
+  if(convexes.count(cName) != 0)
+  {
+    mc_rtc::log::warning("While loading {}, a convex was already provided for collision geometry specified in URDF",
+                         robot);
```

Comment:
I think it's unlikely to show up a lot as it will only appear if:

1. the collision tag is not a mesh; AND
2. there is a single collision tag and the RobotModule already provides a convex for this body; OR
3. the RobotModule has multiple convex hulls that follow the same naming convention as the tag loading

I think this deserves a warning as the user should then choose whether to keep the convex(es) or the collision tag(s)

Reply:
Fair enough, this can remain as-is then :)